src/main/drivers/usb_msc_f4xx.c

```diff
@@ -91,6 +91,14 @@ uint8_t mscStart(void)
 	return 0;
 }
 
+bool mscBoot(void)
```

Comment:
Are these needed for F7?

Reply:
I would say it's cleaner this way as the value stays in different memory region?

Reply:
All good then. I was just wondering if that was a leftover from testing.